workbench-core/environments/src/services/environmentTypeConfigService.test.ts

```diff
@@ -224,14 +244,17 @@ describe('environmentTypeConfigService', () => {
       });
 
       // OPERATE
-      const actualResponse = await envTypeConfigService.createNewEnvironmentTypeConfig(
-        'owner-123',
+      const actualResponse = await envTypeConfigService.createNewEnvironmentTypeConfig({
         envTypeId,
-        createParams
-      );
+        params: createParams
+      });
 
       // CHECK
-      expect(actualResponse).toEqual(envTypeConfig);
+      expect(actualResponse).toEqual({
+        ...envTypeConfig,
+        createdAt: actualResponse.createdAt,
```

Comment:
shouldn't `envTypeConfig` already have `createdAt` and `updatedAt` on it?

Reply:
I was not returning the ddb response but the object created, meaning that created and updated at would be always different, I changed this in the last commit and fixed this issue